internal/backend/runtime/omni/controllers/omni/internal/task/machine/poll.go

```diff
@@ -317,7 +318,7 @@ func pollTalosMachineStatus(ctx context.Context, c *client.Client, info *Info) e
 		}
 
 		return &specs.MachineStatusSpec_TalosMachineStatus{
-			UpdatedAt: timestamppb.New(r.Metadata().Updated()),
+			UpdatedAt: timestamppb.New(time.Now()),
```

Comment:
would it cause endless `MachineStatus` updates?

Reply:
Shouldn't, as we poll only when the watch marks it as dirty. 
But we still will update all resources on startup.

We can probably store the version in the resource instead. And then compare it when we update `MachineStatus`.

Reply:
Added version check. Now it won't do any updates if the resource version isn't increased, that should guarantee that it won't update too frequently.